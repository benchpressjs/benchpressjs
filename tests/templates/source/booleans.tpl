{@false}

{@true}

{!@false}

{!@true}

{(@true == @true)}

{(@false == @true)}

{(@false != @false)}

{(@true != @false)}

{(@true && @true)}

{(@false || @true)}

{(@false || @false)}

{(@true && @false)}
