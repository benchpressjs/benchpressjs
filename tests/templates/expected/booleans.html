false

true

true

false

true

false

false

true

true

true

false

false
